### ISSUE TITLE
"Illegal instruction" when running on NAS (x86_64) Ubuntu 18.04.5 LTS

### ISSUE THREAD

**Issue by sorgoz** [Illumina/Nirvana#47]
Hi,

I'm trying to run Nirvana on a NAS device that has Ubuntu 18.04.5 LTS. Arch = x86_64.

When I start annotation, it outputs the header with names and the beginning of the table, but soon the process terminates with "Illegal instruction" message.

How can I get more info what is wrong exactly?

Dotnet version 5.0.201 (also tried earlier versions - couldn't even build the release with 2.1).

**Comment by sorgoz** [comment]
Thanks for the quick response Michael.

I tried using the precompiled lib for centos6 as you suggested - the error changed to another one (something about not seeing the actual data after decompression), so I went further and compiled BlockCompression from the source.

This new build caused exactly the same initial error - Illegal Instruction.

Then I changed `MARCH` variable in the Makefile to `native`, rebuilt the lib and that solved the problem.
